src/PrestaShopBundle/Service/Hook/HookEvent.php

```diff
@@ -64,7 +64,7 @@ public function getHookParameters()
 
         $sfContainer = SymfonyContainer::getInstance();
         if (!is_null($sfContainer) && !is_null($sfContainer->get('request_stack')->getCurrentRequest())) {
-            $globalParameters['request'] = $sfContainer->get('request');
+            $globalParameters['request'] = $sfContainer->get('request_stack')->getCurrentRequest();
```

Comment:
Same change to apply in `src/Adapter/HookManager`, line 58

https://github.com/mickaelandrieu/PrestaShop/blob/113a589a670aa1af2d1e91b7c0a6ba2e0094526a/src/Adapter/HookManager.php#L58

Reply:
See https://github.com/mickaelandrieu/PrestaShop/pull/9

Reply:
yes we need to apply this change too, `request` service is not available anymore